### PR TITLE
small fixes

### PR DESCRIPTION
**Pull request by nakata5321**
grammar fixes and change image

### CODE REVIEW

File: docs/en/raspberry-setup.md [hunk 1]
```diff
@@ -8,14 +8,14 @@ translated: true
...
+The easiest way to setup you Raspberry is to use our prepared image. You can download it [here.](https://disk.yandex.ru/d/R4vYp9Jhn2O25A) Then read "Configuration RPi" and install image.
 
 
 <robo-wiki-note type="note">Password is "ubuntu".
-User and password for mosquito broker is - user/pass
+User and password for mosquitto broker is - user/pass
 </robo-wiki-note>
 
-If you want to change MQTT user and password use next command:
+If you want to change MQTT user and password use the next command:
```
Comment (Vourhey) [inline docs/en/raspberry-setup.md:18]: use the following command

File: docs/en/raspberry-setup.md [hunk 2]
```diff
@@ -63,9 +62,9 @@ Then find your address in the local network with:
 ip a
 ```
 
-It must look like `192.168.xx.xx` or `172.xx.xx.xx`.
+It should look like `192.168.xx.xx` or `172.xx.xx.xx`.
 
-Then scan the network with your address and zero in the end (also you can use `arp -a`):
+Then scan your network as shown below replacing the last octet of the address with 0. (One may also use arp -a):
```
Comment (Vourhey) [inline docs/en/raspberry-setup.md:67]: add inline code around `arp -a`

**Review by Vourhey — CHANGES_REQUESTED** [review]
Please have a look at grammar for all the rest languages

File: docs/en/raspberry-setup.md (hunk 2 above)
Comment (nakata5321) [reply docs/en/raspberry-setup.md]: solve in next commit

File: docs/en/raspberry-setup.md (hunk 1 above)
Comment (nakata5321) [reply docs/en/raspberry-setup.md]: solve in next commit

**Pull request merged**